src/v/scale/VideoScale.java

```diff
@@ -0,0 +1,48 @@
+package v.scale;
+
+/**
+ * Interface for an object that conveys screen resolution/scaling information, meant to replace the static declarations
+ * in Defines.
+ *
+ * @author admin
```

Comment:
If it was already there the @author should be changed to vlektron (Maes), if you wrote it then it's your name that goes here.